storybookjs/storybook
# (#1286) [WIP] Update package.json to use workspaces in react native examples

Issue:#1286

## Why this PR exists

Currently, our build process requires us to take some additional steps for React Native, which slows the development process. The goal here is to get React Native working with yarn workspaces.

## Checklist

- [ ] Implement Haul in react native vanilla
- [ ] Implement Haul in crna
- [ ] Update package.json in root directory
- [ ] Update package.json in example apps

## What I did

Updated `package.json` in `examples/crna-kitchen-sink` and `examples/react-native-vanilla`

## How to test

Is this testable with jest or storyshots?
Yes (existing tests should cover it)

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.

## Screenshots
